### PR TITLE
add io.github.omrion.puter-electron

### PR DESCRIPTION
**Pull request by OmriOn**
### Please confirm your submission meets all the criteria

Please describe your application briefly below.

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/OmriOn/puter-electron
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by hfiguiere** [comment]
Missing 

> Please describe your application briefly below.

File: io.github.omrion.puter-electron.png
Comment (hfiguiere) [inline io.github.omrion.puter-electron.png:1]: this should be managed upstream

File: io.github.omrion.puter-electron.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  - --filesystem=/run/udev:ro
```
Comment (hfiguiere) [inline io.github.omrion.puter-electron.yaml:19]: why do you need udev access?

File: io.github.omrion.puter-electron.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+modules:
+  - name: puter-electron
+    buildsystem: simple
+    build-commands:
+      - unzip -d puter-electron puter-electron_*.zip
+      - rm puter-electron_*.zip
```
Comment (hfiguiere) [inline io.github.omrion.puter-electron.yaml:26]: with `type: archive` you don't need to do that

File: io.github.omrion.puter-electron.desktop
Comment (hfiguiere) [inline io.github.omrion.puter-electron.desktop:1]: a desktop app without a .desktop file it's a bug. this should be managed upstream.

File: io.github.omrion.puter-electron.desktop [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=Puter
+Comment=The Internet OS!
+Type=Application
+Exec=
```
Comment (hfiguiere) [inline io.github.omrion.puter-electron.desktop:6]: exec can NOT be empty.

**Comment by OmriOn** [comment]
sorrry im kinda new to this can you help me fix thesse

**Pull request closed without merging**